src/insight/check.py

```diff
@@ -106,7 +105,9 @@ def categorical(self, sr: pd.Series) -> bool:
         if pd.api.types.is_categorical_dtype(sr) is True:
             return True
         sr = self.infer_dtype(sr)
-        if sr.dtype.kind == "M":  # TODO: Need to implement ability to deal with dates well in metrics
+        if (
+            sr.dtype.kind == "M"
+        ):  # TODO: Need to implement ability to deal with dates well in metrics
```

Comment:
Rather than have this TODO would it be better to just open an issue on the repo?

Reply:
That comment was there before :smile_cat: I will remove it and open as issue, sure